### ISSUE TITLE
npm compat: Magika

### ISSUE THREAD

**Issue by sigmaSd** [denoland/deno#22432]
Version: Deno 1.40.5

```ts
import { Magika } from "npm:magika";
const magika = new Magika();
await magika.load();
```

```
error: Uncaught (in promise) TypeError: Cannot destructure property 'modelURL' of 'undefined' as it is undefined.
    at Magika.load (file:///home/mrcool/.cache/deno/npm/registry.npmjs.org/magika/0.2.4/magika.js:73:16)
    at file:///home/mrcool/dev/deno/lab/a.ts:7:14
```

**Comment by babiabeo** [comment]
Looks like this isn't a Deno problem because Node.js also has the same error:

```js
import { Magika } from "magika";

const magika = new Magika();
await magika.load();
```

```sh
file:///D:/coding/project/nodejs/mak/node_modules/magika/magika.js:73
  async load({ modelURL, configURL }) {
               ^

TypeError: Cannot destructure property 'modelURL' of 'undefined' as it is undefined.
    at Magika.load (file:///D:/coding/project/nodejs/mak/node_modules/magika/magika.js:73:16)
    at file:///D:/coding/project/nodejs/mak/index.js:4:14
    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at async loadESM (node:internal/process/esm_loader:28:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12)
```

**Comment by sigmaSd** [comment]
The original issue is being tracked here https://github.com/google/magika/issues/55 but this still have issues with deno

```ts
import { Magika } from "npm:magika";

const data = await Deno.readFile("./a.ts");
const magika = new Magika();
await magika.load({});
const prediction = await magika.identifyBytes(data);
console.log(prediction);
```

```
============================
Hi, looks like you are running TensorFlow.js in Node.js. To speed things up dramatically, install our node backend, visit https://github.com/tensorflow/tfjs-node for more details.
============================
error: Uncaught (in promise) RangeError: offset is out of bounds
    at Uint16Array.set (<anonymous>)
    at Magika._extractFeaturesFromBytes (file:///home/mrcool/.cache/deno/npm/registry.npmjs.org/magika/0.2.4/magika.js:177:9)
    at Magika._identifyBytes (file:///home/mrcool/.cache/deno/npm/registry.npmjs.org/magika/0.2.4/magika.js:138:53)
    at Magika.identifyBytes (file:///home/mrcool/.cache/deno/npm/registry.npmjs.org/magika/0.2.4/magika.js:87:17)
    at file:///home/mrcool/dev/deno/lab/a.ts:6:33
    at eventLoopTick (ext:core/01_core.js:153:7)
```

I tested node this time and it does work with the above

**Comment by sigmaSd** [comment]
seems to be fixed in deno 1.41.1

**Comment by drewbitt** [comment]
In your example you were still pulling 0.2.4 magika which was broken. Deno was caching the dependency. You should specify a version `npm:magika@^0.2.5` or reload cache with flag `-r`
